### DISCUSSION TITLE
Multiple Models in one Table?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#8569] [Help]
Is it possible to list multiple Models in one Table? I have a user related to different Models. I would like to List all the related Model in one Table.

**Comment by leandrocfe** [comment]
you mean, like [UNION](https://www.w3schools.com/sql/sql_union.asp)?

**Comment by RPotkowaVegasystems** [comment]
Yes

**Reply by leandrocfe** [reply]
You can create a [custom model](https://stackoverflow.com/questions/26593270/how-to-access-db-views-using-laravel-models), but your resource will be read-only."
